packages/grafana-ui/src/Intro.story.mdx

```diff
@@ -1,12 +1,26 @@
 <Meta title="Docs Overview/Intro" parameters={{ options: { isToolshown: false }, id: 1 }} />
 
-# Grafana UI library
+# Grafana design system
 
 > **@grafana/ui is currently in BETA**.
 
-@grafana/ui is a collection of components used by [Grafana](https://github.com/grafana/grafana)
+With the design system @grafana/ui, we want to democratize development. This library of reusable [Grafana](https://github.com/grafana/grafana) components and guidelines helps you with contribution and plugin development.
 
-If you're a plugin developer, you can use these component to give your plugin an official look-and-feel.
+## A library of unified components and styles
+Grafana-ui is a library of reusable components and guidelines on how to use and develop them. You can arrange them into any complex interface you need, for example in your Grafana plugin, streamlining your design process.
+
+**Our vision**
+Grafana Labs started grafana-ui to make contributing to Grafana as easy as possible for Grafanistas and community members of all fields. We want to create a component library that results in
+1. Understanding of how each component works and how you can use it to create a great user experience. 
+2. Short development times and consistent code quality.
+3. A beautiful, visually consistent Grafana experience.
+4. Transparency about how we work and what we do.
+
+**Maintained by Grafana Labs and you**
+Grafana Labs has a task force that helps create and maintain components. We make sure that components are documented and easy to use. The current status of the grafana-ui development is available on [GitHub](https://github.com/grafana/grafana/projects/26). Feel free to contribute!
+
+**How to get involved**
+When we notice that we need to change something, we ideate on what the change should be, then we put the change in place and communicate it publicly. Developers and designers create and improve grafana-ui together. Throughout the process, we strive to involve you and meet your needs. We are looking forward to discussing your design and improvement ideas on [GitHub](https://github.com/grafana/grafana/projects/26). 
```

Comment:
Not sure if it's just me but `ideate` sounds a bit weird. I'd use smth like `brainstorm`.